### PR TITLE
install jormun requirements during the tyr's upgrade

### PR DESCRIPTION
**Pull request by Hoshiyo**

### CODE REVIEW

File: fabfile/tasks.py [hunk 1]
```diff
@@ -251,6 +251,9 @@ def upgrade_tyr(up_confs=False, pilot_tyr_beat=True, reload=True):
     if pilot_tyr_beat:
         execute(tyr.stop_tyr_beat)
     execute(tyr.upgrade_tyr_packages)
+    # navitia-common is used by tyr and jormungandr
+    # so jormungandr dependancies have to be updated before the apache's restart
+    execute(jormungandr.install_jormun_requirements)
```
Comment (antoine-de) [inline fabfile/tasks.py]: but at this point you won't have upgraded the jormun packages no ?

Comment (Hoshiyo) [inline fabfile/tasks.py]: not jormun but common yes

Comment (antoine-de) [inline fabfile/tasks.py]: but you need jormun there no ?

Comment (Hoshiyo) [inline fabfile/tasks.py]: you are right ...

**Pull request closed without merging**